sig-auth/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to SIG Auth
+
+Welcome to contributing to SIG Auth.
+
+If you haven't seen them already, the Kubernetes project has:
+
+- A [Contributor Guide][contrib-guide] - some
```

Comment:
Missing links here? May borrow from https://github.com/kubernetes/community/blob/master/sig-cluster-lifecycle/CONTRIBUTING.md

Reply:
Updated as per the suggestion.